ojssqla/logic.py

```diff
@@ -1201,3 +1326,30 @@ def mark_reminder_sent(session, review_id, date_sent):
     setattr(assignment, 'date_reminded', date_sent)
     setattr(assignment, 'reminder_was_automatic', 1)
     session.flush()
+
+
+def add_access_key(session, reviewer_id, review_id, new_key_hash):
+    """Add a new access key for an existing review.
+
+    Args:
+        reviewer_id: user ID of reviewer.
+        review_id: ID of the review assignment.
+        new_key_hash: MD5 hashed version of the new access key.
+
+    Returns:
+        The new access key object if successful, None if not.
+    """
+    try:
+        new_access_key_entry = ojs.AccessKey(
+            context='ReviewerContext',
+            key_hash=new_key_hash,
+            user_id=reviewer_id,
+            assoc_id=review_id,
+            expiry_date=datetime.now() + timedelta(weeks=12)
+        )
+        session.add(new_access_key_entry)
+        session.flush()
+        return new_access_key_entry
+
+    except:
```

Comment:
I know this isn't code you wrote, but we should never be allowing global exception catches like this. If we are going to catch exceptions, we should specify which ones we want to catch, and let the others through, thus giving us an indication of what is going wrong, rather than scratching our heads as to why we are getting None as a return value.